src/uikit/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Box,
+  Container,
+  Image,
+  SimpleGrid,
+  Stack,
+  Text,
+} from '@chakra-ui/react';
+import type { ReactNode } from 'react';
+
+function ListHeader({ children }: { children: ReactNode }) {
```

Comment:
`ListHeader` i think better put below the exported component because when we read this file, we can directly know what is this file exporting

Reply:
okay, i guess im gonna move it to separate file, because the linter wont allow me to use component before the component is defined

![image](https://github.com/mocha-bot/land/assets/49477728/29e1e869-b309-48fd-a25d-6452b3a6e67f)

Reply:
or i dont need to separate it at all, just use normal text component as usual because there is not much menu list on footer

Reply:
updated